### PR TITLE
#1765 - replaced 3 try-catch clauses with if-else clauses

### LINKED ISSUE #1765: Remove try-catch from spritecomponent control flow
Replace GetResource with TryGetResource and don't use try-catch at all. It's in 2 spots.

### PR DESCRIPTION
**Pull request by GalacticChimp**
fixes #1765 - replaced 3 try-catch clauses with if-else clause + VS autoformat of affected classes

every change not marked by my comment was made by VS autoformat

### CODE REVIEW

File: Robust.Client/GameObjects/Components/Renderable/SpriteComponent.cs [hunk 1]
```diff
@@ -162,11 +162,12 @@ private List<PrototypeLayerData> LayerDatums
                     if (!string.IsNullOrWhiteSpace(layerDatum.RsiPath))
                     {
                         var path = TextureRoot / layerDatum.RsiPath;
-                        try
+
+                        if (IoCManager.Resolve<IResourceCache>().TryGetResource(path, out RSIResource? resource))
```
Comment (GalacticChimp) [inline Robust.Client/GameObjects/Components/Renderable/SpriteComponent.cs:166]: actual change

File: Robust.Client/GameObjects/Components/Renderable/SpriteComponent.cs [hunk 2]
```diff
@@ -348,13 +349,13 @@ void ISerializationHooks.AfterDeserialization()
                 if (!string.IsNullOrWhiteSpace(rsi))
                 {
                     var rsiPath = TextureRoot / rsi;
-                    try
+                    if(IoCManager.Resolve<IResourceCache>().TryGetResource(rsiPath, out RSIResource? resource))
```
Comment (GalacticChimp) [inline Robust.Client/GameObjects/Components/Renderable/SpriteComponent.cs:352]: actual change

File: Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs [hunk 3]
```diff
@@ -180,20 +180,17 @@ private void LoadLightingShaders()
             _fovCalculationProgram = _compileProgram(depthVert, depthFrag, attribLocations, "Shadow Depth Program");
 
             var debugShader = _resourceCache.GetResource<ShaderSourceResource>("/Shaders/Internal/depth-debug.swsl");
-            _fovDebugShaderInstance = (ClydeShaderInstance) InstanceShader(debugShader.ClydeHandle);
+            _fovDebugShaderInstance = (ClydeShaderInstance)InstanceShader(debugShader.ClydeHandle);
 
             ClydeHandle LoadShaderHandle(string path)
             {
-                try
+                if (_resourceCache.TryGetResource(path, out ShaderSourceResource? resource))
```
Comment (GalacticChimp) [inline Robust.Client/Graphics/Clyde/Clyde.LightRendering.cs:187]: actual change

**Review by GalacticChimp — COMMENTED** [review]
Everything not marked by a comment is a VS autoformat change

**Review by PaulRitter — APPROVED** [review]
good first pr.

fyi, no need to point out each actual change, saves you some time c:

**Comment by PaulRitter** [comment]
another fyi: writing "fixes #1765" autocloses the issue when the pr gets merged

**Pull request merged**